Doc/reference/introduction.rst

```diff
@@ -83,6 +83,11 @@ MicroPython
    possible to allow the transfer of code with ease from the desktop.  For more
    information, see `the MicroPython website <https://micropython.org/>`_.
 
+CircuitPython
+   A MicroPython fork targeted towards students and beginners, with a strong
+   focus on Adafruit boards.  For more information, see `the CircuitPython
```

Comment:
Thanks @andresdelfino. All of Adafruit's boards are open-hardware which would be good to mention.  Perhaps "...on Adafruit's open-hardware boards.

Reply:
Done :)